src/renderer/services/plugin-manager.js

```diff
@@ -620,6 +620,14 @@ class PluginManager {
       sandbox.walletApi.http = new PluginHttp(config.urls)
     }
 
+    if (config.permissions.includes('PEER_CURRENT')) {
+      sandbox.walletApi.peers = {
+        getCurrent: () => {
+          return cloneDeep(this.app.$client.client)
```

Comment:
It's exposing the `typescript-client` instance, is that correct?

I can make any request just by changing the host:

```
const client = walletApi.peers.getCurrent();
client.host = 'https://api.github.com'
const response = await client.get('orgs/arkecosystem')
```

Reply:
Hm did that work for you? `host` is supposed to be [private readonly](https://github.com/ArkEcosystem/typescript-client/blob/master/src/connection.ts#L10). I'll take a look and find a way around it, thanks 👌

Reply:
I guess it's because it's converted to JS, the private readonly is disregarded